### PR TITLE
[FR] Update `Beatmap_Spotlights` and add `Beatmap_Spotlights/Chart_Assembly_Team`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - cat
+  - cats
+  - charts assembly team
+---
+
+# Chart Assembly Team
+
+*Pour la version moderne de l'équipe, voir : [Beatmap Spotlights § Curateurs](/wiki/Beatmap_Spotlights)*
+
+La **Chart Assembly Team** (souvent appelée ***CAT***) était un groupe d'utilisateurs dévoués formé en août 2011[^cat-birth] et responsable de la compilation de beatmaps mensuels et thématiques pour les Ranking Charts (renommés par la suite en [Beatmap Spotlights](/wiki/Beatmap_Spotlights)). Initialement composée des membres de la [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) et de la [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team), elle a été élargie à un ensemble de joueurs et de mappeurs renommés afin d'accroître la diversité.
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md:12]:
```suggestion
La **Chart Assembly Team** (souvent appelée ***CAT***) était un groupe d'utilisateurs dévoués formé en août 2011[^cat-birth] et responsable de la compilation de beatmaps mensuelles et thématiques pour les Ranking Charts (renommés par la suite en [Beatmap Spotlights](/wiki/Beatmap_Spotlights)). Initialement composée des membres de la [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) et de la [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team), elle a été élargie à un ensemble de joueurs et de mappeurs renommés afin d'accroître la diversité.
```

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Over time, there have been several attempts to revitalise the team and increase quality of the Ranking Charts, including two application rounds in 2013[^cat-recruitment-2013] and 2014[^cat-recruitment-2014] and extensive discussions[^cat-hiatus][^cat-crisis] about possible changes to the team format and responsibilities.
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md:14]:
```suggestion
```

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md:15]:
```suggestion
```

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Au fil du temps, il y a eu plusieurs tentatives pour revitaliser l'équipe et améliorer la qualité des Ranking Charts, dont deux cycles de candidatures en 2013[^cat-recruitment-2013] et 2014[^cat-recruitment-2014] et de longues discussions[^cat-hiatus][^cat-crisis] sur les changements possibles du format et des responsabilités de l'équipe.
+
+Le 18 mars 2015, la CAT a été dissoute[^cat-death] en raison de la refonte complète des Ranking Charts, qui comprenait la sélection des maps par la communauté.
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md:18]:
```suggestion
Le 18 mars 2015, la CAT a été dissoute[^cat-death] en raison de la refonte complète des Ranking Charts, qui comprenait la sélection des beatmaps par la communauté.
```

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Références
+
+[^cat-birth]: [(Restreint) Post du forum par Derekku (29/08/2011) dans "MAT Lounge"](https://osu.ppy.sh/community/forums/posts/1004797)
+[^cat-recruitment-2013]: [Fil de discussion par Makar (19/05/2013) "2013 NewCAT Applications"](https://osu.ppy.sh/community/forums/topics/133248)
+[^cat-hiatus]: [(Restreint) Fil de discussion par Ephemeral (17/09/2013) "Ranking charts overhaul"](https://osu.ppy.sh/community/forums/topics/155384)
+[^cat-crisis]: [(Restreint) Fil de discussion par Ephemeral (02/10/2013) "encouraging staff activity + CAT"](https://osu.ppy.sh/community/forums/topics/157800)
+[^cat-recruitment-2014]: [(Restricted) Fil de discussion par Loctav (16/06/2014) "[IMPORTANT] Recruiting NewCATs"](https://osu.ppy.sh/community/forums/topics/218032)
+[^cat-death]: [News post par Loctav (19/03/2015) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md:27]:
```suggestion
[^cat-death]: [Article de news par Loctav (19/03/2015) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
```

File: wiki/Beatmap_Spotlights/fr.md [hunk 6]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
 
 ## Histoire
 
-Nommé à l'origine "Ranking Charts" et lancé en novembre 2009 par ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589) et ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2), le projet visait à mettre en avant les meilleures beatmaps d'un mois en laissant la [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) et la [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) nommer et voter pour les candidats les plus appropriés.
+Nommé à l'origine "Ranking Charts" et lancé en octobre 2009[^charts-09-oct] [^charts-09-nov] par ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) et ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589), le projet visait à mettre en avant les meilleures beatmaps d'un mois ou d'un ans[^charts-10-jan] en laissant la [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) et la [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) nommer et voter pour les candidats les plus appropriés. En septembre 2011, des charts pour osu!taiko et osu!catch sont apparus [^charts-11-per-mode].
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:176]:
```suggestion
Nommé à l'origine "Ranking Charts" et lancé en octobre 2009[^charts-09-oct] [^charts-09-nov] par ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) et ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589), le projet visait à mettre en avant les meilleures beatmaps d'un mois ou d'une année[^charts-10-jan] en laissant la [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) et la [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) nommer et voter pour les candidats les plus appropriés. En septembre 2011, des charts pour osu!taiko et osu!catch sont apparus [^charts-11-per-mode].
```

File: wiki/Beatmap_Spotlights/fr.md [hunk 7]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
...
+En janvier 2014, le navigateur charts a été ajouté au client osu!,[^charts-in-osu-14-jan] et en juin 2014, plusieurs joueurs renommés ont été recrutés dans la [Chart Assembly Team](/wiki/Beatmap_Spotlights/Chart_Assembly_Team).[^charts-cat-recruitment-14-jun] En mars 2015, le projet a changé de son design original de nomination et de vote en ayant des membres renommés de la communauté qui sélectionnent seuls une liste de sets de beatmap qu'ils recommandent.[^charts-curated-15-mar] En septembre 2016, le système de sélection a été en grande partie inversé[^charts-reverted-16-sep] et a mis la [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) en charge de la sélection des beatmaps les plus remarquables.
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:182]:
```suggestion
En janvier 2014, le navigateur charts a été ajouté au client osu!,[^charts-in-osu-14-jan] et en juin 2014, plusieurs joueurs renommés ont été recrutés dans la [Chart Assembly Team](/wiki/Beatmap_Spotlights/Chart_Assembly_Team).[^charts-cat-recruitment-14-jun] En mars 2015, le projet a changé de son design originel de nomination et de vote en ayant des membres renommés de la communauté qui sélectionnent seuls une liste de sets de beatmaps qu'ils recommandent.[^charts-curated-15-mar] En septembre 2016, le système de sélection a été en grande partie inversé[^charts-reverted-16-sep] et a mis la [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) en charge de la sélection des beatmaps les plus remarquables.
```

File: wiki/Beatmap_Spotlights/fr.md [hunk 8]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
...
+
+Renommé Beatmap Spotlights en mars 2017,[^charts-renamed-into-spotlights] le système lui-même est resté globalement cohérent tout en ajoutant des récompenses supplémentaires comme des médailles et en améliorant encore la présentation des Beatmap Spotlights. Lors d'une refonte interne de la Quality Assurance Team, la responsabilité du projet a été réattribuée à ::{ flag=HU }:: [Kurokami](https://osu.ppy.sh/users/260933) et a réimplanté une équipe de sélection basée sur la communauté. En novembre 2018, la fréquence des Spotlights a été modifiée pour devenir un cycle de publication saisonnier.[^spotlights-seasonal] En mars 2020, ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) a repris la direction du projet avec Kurokami, en lui donnant une nouvelle forme et en constituant une nouvelle équipe de curateurs osu!.[^spotlights-reworked-20-june]
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:184]:
```suggestion
Renommé Beatmap Spotlights en mars 2017,[^charts-renamed-into-spotlights] le système lui-même a globalement continué de fonctionner de la même manière tout en ajoutant des récompenses supplémentaires comme des médailles et en améliorant encore la présentation des Beatmap Spotlights. Lors d'une refonte interne de la Quality Assurance Team, la responsabilité du projet a été réattribuée à ::{ flag=HU }:: [Kurokami](https://osu.ppy.sh/users/260933) et a réimplanté une équipe de sélection basée sur la communauté. En novembre 2018, la fréquence des Spotlights a été modifiée pour devenir un cycle de publication saisonnier.[^spotlights-seasonal] En mars 2020, ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) a repris la direction du projet avec Kurokami, en lui donnant une nouvelle forme et en constituant une nouvelle équipe de curateurs osu!.[^spotlights-reworked-20-june]
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Spotlights/fr.md [hunk 9]
```diff
@@ -136,13 +134,13 @@ Ce tableau ne présente qu'une seule des quatre versions de badge. Chaque mode d
 
 Le système de curation implique un processus de sélection par les curateurs des beatmaps mises en avant dans les Beatmap Spotlights pour chaque saison.
 
-1. Les beatmaps sont sélectionnées par les curateurs de leur mode de jeu respectif pour la durée d'une saison.
+1. Les beatmaps sont sélectionnées par les curateurs de leur modes de jeu respectif pour la durée d'une saison.
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:137]:
```suggestion
1. Les beatmaps sont sélectionnées par les curateurs de leur mode de jeu respectif pour la durée d'une saison.
```

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+Le 18 mars 2015, la CAT a été dissoute[^cat-death] en raison de la refonte complète des Ranking Charts, qui comprenait la sélection des beatmaps par la communauté.
+
+## Références
+
+[^cat-birth]: [(Restreint) Post du forum par Derekku (29/08/2011) dans "MAT Lounge"](https://osu.ppy.sh/community/forums/posts/1004797)
+[^cat-recruitment-2013]: [Fil de discussion par Makar (19/05/2013) "2013 NewCAT Applications"](https://osu.ppy.sh/community/forums/topics/133248)
+[^cat-hiatus]: [(Restreint) Fil de discussion par Ephemeral (17/09/2013) "Ranking charts overhaul"](https://osu.ppy.sh/community/forums/topics/155384)
+[^cat-crisis]: [(Restreint) Fil de discussion par Ephemeral (02/10/2013) "encouraging staff activity + CAT"](https://osu.ppy.sh/community/forums/topics/157800)
+[^cat-recruitment-2014]: [(Restricted) Fil de discussion par Loctav (16/06/2014) "[IMPORTANT] Recruiting NewCATs"](https://osu.ppy.sh/community/forums/topics/218032)
+[^cat-death]: [Article de news par Loctav (19/03/2015) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/fr.md:25]:
```suggestion
[^cat-death]: [Article de news par Loctav (19/03/2015) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)

```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Spotlights/fr.md [hunk 11]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
...
+[^charts-manager-sg]: [Fil de discussion par Cyclone (20/05/2012) "May 2012 Ranking Chart, and new Chart Manager"](https://osu.ppy.sh/community/forums/topics/84573)
+[^charts-manager-dxs]: [Fil de discussion par DeathxShinigami (14/04/2013) "April 2013 Ranking Chart"](https://osu.ppy.sh/community/forums/topics/127847)
+[^charts-manager-makar]: [Fil de discussion par Makar (19/05/2013) "2013 NewCAT Applications"](https://osu.ppy.sh/community/forums/topics/133248)
+[^charts-manager-loctav]: [(Restreint) Post du forum par Loctav (21/11/2013) dans "Regarding the charts"](https://osu.ppy.sh/community/forums/posts/2697871)
+[^charts-in-osu-14-jan]: [(Restreint) Post du forum par peppy (24/01/2014) dans "So peppy's planning something chart related"](https://osu.ppy.sh/community/forums/posts/2824323)
+[^charts-cat-recruitment-14-jun]: [(Restreint) Fil de discussion par Loctav (16/06/2014) "\[IMPORTANT\] Recruiting NewCATs"](https://osu.ppy.sh/community/forums/topics/218032)
+[^charts-curated-15-mar]: [News post par Loctav (18/03/2015) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:203]:
```suggestion
[^charts-curated-15-mar]: [Article de news par Loctav (18/03/2015) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
```

File: wiki/Beatmap_Spotlights/fr.md [hunk 12]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
...
+[^charts-reverted-16-sep]: [News post par OnosakiHito (17/09/2016) "July 2016 Ranking Charts - Changes"](https://osu.ppy.sh/home/news/2016-09-17-july-2016-ranking-charts-changes)
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:204]:
```suggestion
[^charts-reverted-16-sep]: [Article de news par OnosakiHito (17/09/2016) "July 2016 Ranking Charts - Changes"](https://osu.ppy.sh/home/news/2016-09-17-july-2016-ranking-charts-changes)
```

File: wiki/Beatmap_Spotlights/fr.md [hunk 13]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
...
+
+[^charts-renamed-into-spotlights]: [News post par OnosakiHito (18/03/2017) "Introducing to you: Spotlights"](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:206]:
```suggestion
[^charts-renamed-into-spotlights]: [Article de news par OnosakiHito (18/03/2017) "Introducing to you: Spotlights"](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
```

File: wiki/Beatmap_Spotlights/fr.md [hunk 14]
```diff
@@ -175,14 +173,36 @@ Tous les candidats sont examinés par la direction du projet et les chefs d'équ
...
+[^spotlights-seasonal]: [News post par Kurokami (01/11/2018) "Seasonal Spotlights: Summer 2018"](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)
```
Comment (Ganondorf-1) [inline wiki/Beatmap_Spotlights/fr.md:207]:
```suggestion
[^spotlights-seasonal]: [Article de news par Kurokami (01/11/2018) "Seasonal Spotlights: Summer 2018"](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]
Can be merged once this is applied

**Review by TicClick — APPROVED** [review]

**Pull request merged**